src/components/v5/frame/ColonyHome/partials/ReputationChart/ReputationChart.tsx

```diff
@@ -1,91 +1,68 @@
-import LegendItem from './partials/LegendItem.tsx';
-import NoTeamsWithReputation from './partials/NoTeamsWithReputation.tsx';
-import { type ReputationChartDataItem } from './types.ts';
+import { useContributorsByDomain } from './hooks/useContributorsByDomain.tsx';
+import { Chart } from './partials/Chart.tsx';
 import {
+  getContributorReputationChartData,
   getTeamReputationChartData,
   isThereReputationInDomains,
 } from './utils.ts';
 
 const displayName = 'v5.frame.ColonyHome.ReputationChart';
 
 const MSG = defineMessages({
-  reputationTitle: {
-    id: `${displayName}.influenceTitle`,
+  reputationTitleTeam: {
+    id: `${displayName}.influenceTitleTeam`,
     defaultMessage: 'Influence by team',
   },
+  reputationTitleContributors: {
+    id: `${displayName}.influenceTitleContributors`,
+    defaultMessage: 'Top contributors',
+  },
 });
 
-const TemporaryTooltip: FC<PieTooltipProps<ReputationChartDataItem>> = ({
-  datum: { label, value },
-}) => {
-  return (
-    <div className="bg-base-black p-2 text-base-white">
-      {label}
-      (<Numeral value={value.toFixed(1)} />
-      %)
-    </div>
-  );
-};
-
 const ReputationChart = () => {
   const {
     colony: { domains },
   } = useColonyContext();
 
+  const { contributorsList } = useContributorsByDomain();
+
+  const selectedDomain = useGetSelectedDomainFilter();
+
   const allTeams = (domains?.items || [])
     .filter(notNull)
     .sort(
       (a, b) => Number(b.reputationPercentage) - Number(a.reputationPercentage),
     );
-  const chartData = getTeamReputationChartData(allTeams);
 
-  const getChartAndLegend = () => {
-    if (chartData.length === 0 || !isThereReputationInDomains(allTeams)) {
-      return <NoTeamsWithReputation />;
-    }
+  const chartDataTeams = !isThereReputationInDomains(allTeams)
+    ? []
+    : getTeamReputationChartData(allTeams);
 
-    return (
-      <>
-        <div className="mb-5 mt-3 flex h-[140px] w-full flex-shrink-0 items-center justify-center">
-          <ResponsivePie
-            {...pieChartConfig}
-            data={chartData}
-            tooltip={TemporaryTooltip}
-          />
-        </div>
-        <Legend>
-          {chartData.map((chartItem) => {
-            // if there is no value, it's value doesn't display in the chart and therefore it shouldn't display in the legend
-            if (chartItem.value === undefined || chartItem.value <= 0) {
-              return null;
-            }
+  const chartDataContributors =
+    getContributorReputationChartData(contributorsList);
 
-            return <LegendItem key={chartItem.id} chartItem={chartItem} />;
-          })}
-        </Legend>
-      </>
-    );
-  };
+  // if domain/team is selected we should show Top Contributors chart instead
+  const chartData = selectedDomain ? chartDataContributors : chartDataTeams;
+  const reputationTitle = selectedDomain
+    ? MSG.reputationTitleContributors
+    : MSG.reputationTitleTeam;
```

Comment:
As I have re-tested, I realised we should actually show the top contributors if the selected domain has no sub-teams, thus `General` should show the teams data
So, in order to fix this and support future nested teams, let's create a helper method 
```
import { type Domain } from '~types/graphql.ts';
const hasNoSubDomains = (domain?: Domain) => 
  !!domain && !domain?.isRoot;
```

and then use it here 
```suggestion
  const chartData = hasNoSubDomains(selectedDomain) ? chartDataContributors : chartDataTeams;
  const reputationTitle = hasNoSubDomains(selectedDomain)
    ? MSG.reputationTitleContributors
    : MSG.reputationTitleTeam;
```